x-pack/plugins/cases/server/authorization/authorization.ts

```diff
@@ -231,8 +231,10 @@ export class Authorization {
           ? Array.from(featureCaseOwners)
           : privileges.kibana.reduce<string[]>((authorizedOwners, { authorized, privilege }) => {
               if (authorized && requiredPrivileges.has(privilege)) {
-                const owner = requiredPrivileges.get(privilege)!;
-                authorizedOwners.push(owner);
+                const owner = requiredPrivileges.get(privilege);
+                if (owner) {
```

Comment:
Remove the `!` non-null assertion operation and check if is empty before pushing.